tests/foreman/cli/test_remoteexecution.py

```diff
@@ -134,6 +134,42 @@ def test_positive_run_default_job_template(
         assert 'Exit' in out
         assert 'Internal Server Error' not in out
 
+    @pytest.mark.tier3
+    @pytest.mark.rhel_ver_list([8])
+    def test_positive_timeout_to_kill(self, module_org, rex_contenthost, module_target_sat):
+        """Use timeout to kill setting to cancel the job
+
+        :id: 580f886b-ac24-4afa-9aca-e9afc5cfdc9c
+
+        :expectedresults: Verify the job was killed after specified times
+
+        :parametrized: yes
+
+        :Verifies: SAT-25243
+        """
+        client = rex_contenthost
+        command = r'for i in \`seq 160\`; do sleep \$i; done'
```

Comment:
(non-blocking for the test functionality)
With `sleep $i`, the sleeps will be incremental, i.e. sleep 1; sleep 2; sleep 3, etc. 
For sleeping for 160 seconds in 1-second ticks, running `for run in {1..160}; do sleep 1; done` will do the trick.